### DISCUSSION TITLE
How to use 'Omen key' for something usable?

### DISCUSSION THREAD

**Discussion by bktech2021** [i3/i3#5913] [Q&A] [answered]
My Victus16 has something called an 'Omen key' which used to open Omen Manager app on Windows. Since I can't install this app at Arch, can I do something usable with id like binding it to Dmenu? 

(Maybe button is binded to something like F13-F24, IDK because i can't check. Most keyboard tools provide F1-F12 keys, and when I click to the button, it says left click which is definietly not, doesnt works like a left click also)

**Comment by orestisfl** [comment, marked as answer]
You can find out the id of the key running `xev -event keyboard`

**Reply by bktech2021** [reply]
```
KeyPress event, serial 37, synthetic NO, window 0x3200001,
    root 0x63e, subw 0x0, time 3117957, (-1,565), root:(963,589),
    state 0x10, keycode 157 (keysym 0x1008ff42, XF86Launch2), same_screen YES,
    XLookupString gives 0 bytes:
    XmbLookupString gives 0 bytes:
    XFilterEvent returns: False
```
I assume its on keycode 157. The `bindcode 157 exec <something>` would work right?

**Reply by bktech2021** [reply]
Yep, it works. Thanks.
